audioread/ffdec.py

```diff
@@ -73,6 +75,18 @@ def run(self):
                 # Stream closed (EOF).
                 break
 
+class WriterThread(threading.Thread):
+    """A thread that writes data to a filehandle
+    """
+    def __init__(self, fh, audio=None):
+        super(WriterThread, self).__init__()
+        self.fh = fh
+        self.audio = audio
+        self.daemon = True
+
+    def run(self):
+        self.fh.write(self.audio.read())
```

Comment:
I'll add block size as an argument similar to `QueueReaderThread` and read a block at a time. Perhaps in a `while True:` loop.